android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -660,65 +660,70 @@ private Boolean installClient() {
      * @return Boolean success
      */
 	private Boolean installFile(String file, Boolean override, Boolean executable) {
-    	Boolean success = false;
-    	byte[] b = new byte [1024];
-		int count; 
-		
-		// If file is executable, cpu architecture has to be evaluated
-		// and assets directory select accordingly
-		String source = "";
-		if(executable) source = getAssestsDirForCpuArchitecture() + file;
-		else source = file;
-		
-		try {
-			if(Logging.ERROR) Log.d(Logging.TAG, "installing: " + source);
+        Boolean success = false;
+        byte[] b = new byte [1024];
+        int count;
+		
+        // If file is executable, cpu architecture has to be evaluated
+        // and assets directory select accordingly
+        String source = "";
```

Comment:
I'm not sure this variable need to be initialized with default value

Reply:
You're right. Fixed.